sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -68,6 +70,11 @@ class StreamingQueryManager private[sql] (sparkSession: SparkSession) extends Lo
         logInfo(s"Registered listener ${listener.getClass.getName}")
       })
     }
+    if (sparkSession.sparkContext.conf.get(UI_ENABLED)) {
+      val statusListener = new StreamingQueryStatusListener(sparkSession.sqlContext.conf)
+      addListener(statusListener)
+      sparkSession.sparkContext.ui.foreach(new StreamingQueryTab(statusListener, _))
```

Comment:
Maybe we should add this `StreamingQueryTab` in `SharedState`, otherwise it will generate one tab for each query. like this:
![image](https://user-images.githubusercontent.com/7402327/71948033-b0c4bf80-3209-11ea-98e8-444e3368a573.png)

Reply:
Thanks for checking, fix in 5accc93.